<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>WebPageLinkDlgClass</class>
 <widget class="QDialog" name="WebPageLinkDlgClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WebPageLinkDlg</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QTextEdit" name="textEdit" >
     <property name="sizePolicy" >
      <sizepolicy>
       <hsizetype>7</hsizetype>
       <vsizetype>5</vsizetype>
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize" >
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="readOnly" >
      <bool>true</bool>
     </property>
     <property name="html" >
      <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;/head>&lt;body style=" white-space: pre-wrap; font-family:宋体; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Select The URL(s) you Wanted and Clicked OK.&lt;/p>&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">You Can Edit the Selection By some Buttoms Below for Convenient.&lt;/p>&lt;/body>&lt;/html></string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="wpl_lw_url" >
     <property name="mouseTracking" >
      <bool>true</bool>
     </property>
     <property name="contextMenuPolicy" >
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="selectionBehavior" >
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="wpl_pb_option" >
       <property name="text" >
        <string>Option</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wpl_pb_mark_highlight" >
       <property name="text" >
        <string>&amp;Mark Highlight</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="wpl_pb_choose" >
       <property name="text" >
        <string>Choose</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_4" >
       <property name="text" >
        <string>&amp;OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_4</sender>
   <signal>clicked()</signal>
   <receiver>WebPageLinkDlgClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>365</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel" >
     <x>325</x>
     <y>338</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>WebPageLinkDlgClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>459</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel" >
     <x>526</x>
     <y>356</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
